recipes/lleaves/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "lleaves" %}
+{% set version = "0.1.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/lleaves-{{ version }}.tar.gz
+  sha256: a2c1d1779a27c316b6d5838c0972d584bf0a39fa658442d280cd78aaa4cc28f5
+
+build:
+  number: 0
+  skip: True  # [win]
```

Why are you skipping Windows? Looks like this could rather be `noarch: python`.
```suggestion
  noarch: python
```